File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    a=s.split()
    a=list(map(int,a))
    b=min(a)
    c=max(a)
    #answer=("%d %d" %(b,c))
    #print(answer)
    return str(b)+" "+str(c)
        
            

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String[] strArr = s.split(" ");
        List<Integer> slist = new ArrayList<>();
        for (String num : strArr) {
            slist.add(Integer.parseInt(num));
        }
        Collections.sort(slist);
        
        int slistLen = slist.size() - 1;
        int max_value = slist.get(slistLen);
        int min_value = slist.get(0);
        
        return Integer.toString(min_value) + " " + Integer.toString(max_value);
    }
}